Project.toml

```diff
@@ -21,28 +24,29 @@ SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 StrideArrays = "d1fa6d79-ef01-42a6-86c9-f7c551f8593b"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
 
 [compat]
```

Comment:
to tag a new version *ALL* packages we depend on must have upper compat bounds, otherwise we are not allowed to register a new version the `General` registry.